static/app/views/releases/detail/index.tsx

```diff
@@ -143,12 +143,15 @@ class ReleasesDetail extends AsyncView<Props, State> {
         query: {
           project: location.query.project,
           environment: location.query.environment ?? [],
-          query: `release:"${params.release}"`,
+          query: searchReleaseVersion(params.release),
           field: 'sum(session)',
           statsPeriod: '90d',
           interval: '1d',
         },
       },
+      {
+        allowError: error => error.status === 400,
```

Comment:
allowing errors from this response, page shouldn't error when it fails. should still render other data